Production/govuk_ios/Views/Home/WidgetView.swift

```diff
@@ -35,6 +32,7 @@ class WidgetView: UIView {
             layer.borderWidth = 1
             layer.cornerRadius = 10
             layer.masksToBounds = true
+            layoutMargins = .init(all: 16)
```

Comment:
This change means that a default, non-zero value for layoutMargins is used if decorateView is false, resulting in the right and left margins of the Topics cards being indented to far (can be seen in updated snapshots). 

I might bring back the `padding` var and use that like so
```
        if decorateView {
            layer.borderWidth = 1
            layer.cornerRadius = 10
            layer.masksToBounds = true
        }
        layoutMargins = .init(all: padding)
```
However, there may be cleaner solutions